### ISSUE TITLE
Carousel: accessibility improvements

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#969]
- carousel requests
  - move controls to the first of the order
  - currently all items tab through
    - only tab through visible cards
  - pagination controls are nice to have
  - don’t focus the whole carousel, just jump to the first item

**Comment by jaketrent** [comment]
I believe the pagination controls should be removed from this bug issue at the least.  Their addition was definitely mentioned in our a11y discussion with Amber, but my sense was that she clearly put them in the box of nice-to-have.  I think we should separately explore with her why they'd be nice, what the potential value-add is, and then make an ROI decision on the work involved there.
